service/src/main/java/cz/muni/fi/pa165/facade/PersonFacadeImpl.java

```diff
@@ -21,7 +21,6 @@
  * @date 14/11/2016
  */
 @Service
-@Transactional
```

Comment:
Urcite to musi byt pri vsetkych Facade, ak crashuju testy, chyba bude niekde inde.